### PR TITLE
Update 10min.ipynb with addition Dask verbiage/clarification

### PR DESCRIPTION
**Pull request by taureandyernv**
Adds additional requested Dask docs related to #2277.  

The main update has us add this sentence
```
For instance, when you call dask_cudf.read_csv(...), your cluster’s GPUs do the work of parsing the CSV file(s) with underlying cudf.read_csv().
```

Thanks @randerzander

### CODE REVIEW

**Comment by jangorecki** [comment]
@taureandyernv #2277 was about docstring so I think this PR does not address that issue. It is of course useful anyway, but IMO not sufficient to resolve #2277.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # cuDF 0.12.0 (Date TBD)
 
 ## New Features
-
+- PR #3759 Updated 10 Minutes with clarification on how `dask_cudf` uses `cudf` API
```
Comment (kkraus14) [inline CHANGELOG.md]: Formatting
```suggestion

- PR #3759 Updated 10 Minutes with clarification on how `dask_cudf` uses `cudf` API
```

File: docs/cudf/source/10min.ipynb [hunk 2]
```diff
@@ -4,24 +4,24 @@
...
+        "\n",
+        "Modeled after 10 Minutes to Pandas, this is a short introduction to cuDF and Dask-cuDF, geared mainly for new users.\n",
+        "\n",
+        "### What are these Libraries?\n",
+        "\n",
+        "[cuDF](https://github.com/rapidsai/cudf) is a Python GPU DataFrame library (built on the Apache Arrow columnar memory format) for loading, joining, aggregating, filtering, and otherwise manipulating tabular data using a DataFrame style API.\n",
+        "\n",
+        "[Dask](https://dask.org/) is a flexible library for parallel computing in Python that makes scaling out your workflow smooth and simple. On the CPU, Dask uses Pandas to execute operations in parallel on DataFrame partitions.\n",
+        "\n",
+        "[Dask-cuDF](https://github.com/rapidsai/dask-cudf) is extends Dask where necessary to allow its DataFrame partitions to be processed by cuDF GPU DataFrames. For instance, when you call dask_cudf.read_csv(...), your cluster’s GPUs do the work of parsing the CSV file(s) with underlying cudf.read_csv().\n",
```
Comment (kkraus14) [inline docs/cudf/source/10min.ipynb]:
```suggestion
        "[Dask-cuDF](https://github.com/rapidsai/dask-cudf) extends Dask where necessary to allow its DataFrame partitions to be processed by cuDF GPU DataFrames as opposed to Pandas DataFrames. For instance, when you call dask_cudf.read_csv(...), your cluster’s GPUs do the work of parsing the CSV file(s) with underlying cudf.read_csv().\n",
```

**Comment by harrism** [comment]
Waiting on author,  end of burn down, so pushing to 0.13.
@taureandyernv

**Comment by taureandyernv** [comment]
@harrism sorry for the delay.  i approved the suggestions and it is ready for review and commit.  can it go back into 0.12?

**Comment by harrism** [comment]
@kkraus14 can this be approved?

**Comment by kkraus14** [comment]
Thanks @taureandyernv for putting this in! 😄

**Pull request merged**